packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -275,6 +275,9 @@ enum GestureRecognizerState {
   /// been accepted definitively.
   possible,
 
+  /// The gesture has been definitively accepted by the recognizer.
+  accepted,
```

Comment:
i am not convinced about adding this. I don't really understand what it gives us.